tutorials/3d/sdfgi.rst

```diff
@@ -0,0 +1,245 @@
+- **Normal Bias:** The normal bias to use for SDFGI's probe ray bounces.
+  Unlike **Probe Bias**, this only increases the value in relation to the
+  mesh's normals. This makes the bias adjustment more nuanced and avoids
+  increasing the bias too much for no reason. Increase this
+  value if you notice striping artifacts in indirect lighting or reflections.
+- **Probe Bias:** The bias to use for SDFGI's probe ray bounces. Increase this
+  value if you notice striping artifacts in indirect lighting or reflections.
+
+SDFGI interaction with lights and objects
+-----------------------------------------
+
+The amount of indirect energy emitted by a light is governed by its color,
+energy *and* indirect energy properties. To make a specific light emit more
+or less indirect energy without affecting the amount of direct light emitted
+by the light, adjust the **Indirect Energy** property in the Light3D inspector.
+
+To ensure correct visuals when using SDFGI, you must configure your meshes
+and lights' global illumination properties according to their *purpose* in the
+scene (static or dynamic).
+
+There are 3 global illumination modes available for meshes:
+
+- **Disabled:** The mesh won't be taken into account in SDFGI generation.
+  The mesh will receive indirect lighting from the scene, but it will not
+  contribute indirect lighting to the scene.
+- **Static (default):** The mesh will be taken into account in VoxelGI generation. The mesh will
+  both receive *and* contribute indirect lighting to the scene. If the mesh
+  is changed in any way after SDFGI is generated, the camera must move away from
+  the object then move back close to it for SDFGI to regenerate. Alternatively,
+  SDFGI can be toggled off and back on. If neither is done, indirect lighting
+  will look incorrect.
+- **Dynamic (not supported with SDFGI):** The mesh won't be taken into account in SDFGI generation.
+  The mesh will receive indirect lighting from the scene, but it will not
+  contribute indirect lighting to the scene.
+  *This acts identical to the **Disabled** bake mode when using SDFGI.*
+
+Additionally, there are 3 bake modes available for lights
+(DirectionalLight3D, OmniLight3D and SpotLight3D):
+
+- **Disabled:** The light won't be taken into account for VoxelGI baking.
+  The light won't contribute indirect lighting to the scene.
+- **Static:** The light will be taken into account for VoxelGI baking.
+  The light will contribute indirect lighting to the scene. If the light
+  is changed in any way after baking, the VoxelGI node must be baked again or indirect lighting
+  will look incorrect. If in doubt, use this mode for level lighting.
+- **Dynamic (default):** The light won't be taken into account for VoxelGI baking,
+  but it will still contribute indirect lighting to the scene in real-time.
+  This option is slower compared to **Static**. Only use the **Dynamic** global
+  illumination mode on lights that will change significantly during gameplay.
```

Comment:
This looks tailored towards VoxelGI, not SDFGI. Is this a copy-paste error?

Reply:
Indeed, I've noticed that SDFGI light behaves differently depending on whether the bake mode is Static or Dynamic. The Static mode will indeed prevent updates once the light has entered the cascade.

Performance seems to be mostly identical from my testing, and I'm not sure if quality is actually better with lights in Static mode.